.github/workflows/deploy.yml

```diff
@@ -206,18 +206,18 @@ jobs:
 
       - name: Deploy Kurtosis CDK package with additional services
         run: |
-          yq -Y --in-place '.args.additional_services = ["blockscout", "blutgang", "pless_zkevm_node", "prometheus_grafana", "tx_spammer"]' params.yml
+          yq -Y --in-place '.args.additional_services = ["arpeggio", "blockscout", "blutgang", "pless_zkevm_node", "prometheus_grafana", "tx_spammer"]' params.yml
           kurtosis run --enclave=${{ env.ENCLAVE_NAME }} --show-enclave-inspect=false --args-file=params.yml .
 
       - name: Inspect enclave
         run: kurtosis enclave inspect ${{ env.ENCLAVE_NAME }}
 
       - name: Verify Blutgang rpc
         run: cast bn --rpc-url $(kurtosis port print ${{ env.ENCLAVE_NAME }} blutgang-001 http)
```

Comment:
Can you add a similar check in the `additional_services` job, to make sure arpeggio is correctly working?

```yml
- name: Verify Arpeggio RPC
   run: cast bn --rpc-url $(kurtosis port print ${{ env.ENCLAVE_NAME }} arpeggio-001 rpc)
```

Reply:
added